tests/models/prophetnet/test_modeling_prophetnet.py

```diff
@@ -436,10 +436,10 @@ def check_fast_integration(
                 decoder_attention_mask=decoder_attention_mask,
                 labels=lm_labels,
             )
-        self.parent.assertTrue(torch.allclose(result.loss, torch.tensor(4.5819, device=torch_device), atol=1e-3))
+        self.parent.assertTrue(torch.allclose(result.loss, torch.tensor(4.5981, device=torch_device), atol=1e-3))
```

Comment:
Hard-coded values on a random model, which is now initialized differently with the only one init rule.